src/server/authserver/Server/AuthSession.h

```diff
@@ -23,6 +23,7 @@
 #include "ByteBuffer.h"
 #include "Socket.h"
 #include "BigNumber.h"
+#include "QueryCallback.h"
```

Comment:
Including QueryCallback is not the proper way to do this - it is not used anywhere in this file. I have a strong feeling that the error is something else but including QueryCallback.h indirectly includes the required header.